docs/guide/conditional-fields.md

```diff
@@ -30,20 +33,231 @@ add: {
     type: 'integer',
     min: 1,
     max: 5,
-    // 👇 Here is our condition definition
+    // 👇 Here is our simple condition definition
     if: {
       seenMovie: true
     }
   }
 }
 ```
+
+## MongoDB-style comparison operators
+
+For more flexible conditions, you can use MongoDB-style comparison operators instead of exact value matching. This allows for range checks, existence checks, and matching against multiple values.
+
+```javascript
+// A field schema's `add` configuration
+add: {
+  priority: {
+    label: 'Task Priority',
+    type: 'select',
+    choices: [
+      { label: 'Low', value: 'low' },
+      { label: 'Medium', value: 'medium' },
+      { label: 'High', value: 'high' },
+      { label: 'Critical', value: 'critical' }
+    ]
+  },
+  estimatedHours: {
+    label: 'Estimated Hours',
+    type: 'integer'
+  },
+  assignee: {
+    label: 'Assignee',
+    type: 'string'
+  },
+  escalationNotes: {
+    label: 'Escalation Notes',
+    type: 'string',
+    textarea: true,
+    // 👇 Show this field for high/critical priority tasks with 20+ hours
+    if: {
+      priority: { $in: ['high', 'critical'] },
+      estimatedHours: { $gte: 20 },
+      assignee: { $exists: true }
```

Comment:
The `$exists` operator is tricky. It does what it should but I think Apostrophe sets default empty values by type and in this case `assignee` is most probably an empty string and this evaluates always to true.

The proper way to do it is `'assignee.length': { $gt: 0 }`.